.github/workflows/triage.yml

```
@@ -57,6 +57,21 @@ jobs:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: gh pr edit --add-label workflows '${{ github.event.number }}'
 
+  limit-pull-requests:
+    if: always() && github.repository_owner == 'Homebrew'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: Homebrew/actions/limit-pull-requests@master
+        with:
+          except-users: |
+            BrewTestBot
+          comment-limit: 15
+          comment: |
+            Hi, thanks for your contribution to Homebrew! You already have 15 open pull requests, so please wait for some to be merged or closed before you open more. If CI fails on any of them, please have a look and try to fix them if you can.
+            If you are performing simple version bumps, we have a bot that automatically bumps [a list of formulae](https://github.com/Homebrew/homebrew-core/blob/HEAD/.github/autobump.txt) so you don't need to. Please take a look at Homebrew/homebrew-core#139929, or any other issues / pull requests labelled https://github.com/Homebrew/homebrew-core/labels/help%20wanted, and see if you can help to fix any of them. Thanks!
```

Would make this text easier to copy to different repos
```suggestion
            If you are performing simple version bumps, we have a bot that automatically bumps [a list of formulae](https://github.com/${{ gitbub. repository }}/blob/HEAD/.github/autobump.txt) so you don't need to. Please take a look at Homebrew/homebrew-core#139929, or any other issues / pull requests labelled https://github.com/${{ gitbub. repository }}/labels/help%20wanted, and see if you can help to fix any of them. Thanks!
```